### DISCUSSION TITLE
[1 febbraio 2022 - A] Esercizio 1

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#205] [Esami]
**Esercizio 1 (7 punti):**

Dare la rappresentazione tabellare di un automa che riceve in input una sequenza di bit e dà in output 1 ogni volta che gli ultimi 3 bit, intesi come un intero in Ca2, rappresentano un numero che sia o negativo oppure multiplo di 4, anche con sovrapposizioni; quando non sono stati ricevuti ancora 3 bit, si restituisca 0. Si modifichi poi l’automa in modo da non ammettere sovrapposizioni e si sintetizzi il circuito associato (fino alle espressioni booleane minime), usando un FF di tipo SR per il bit di stato più significativo e FF di tipo T per i restanti bit. Realizzare la parte combinatoria utilizzando una PLA.

**Comment by Elia-Belli** [comment]
Stesso esercizio dell'Esame B dello stesso appello (#211 )
